### PR TITLE
De warp net review

### PR DESCRIPTION
**Pull request by Rajatava**

### CODE REVIEW

File: DewarpNet_Single_Image_Document_Unwarping_With_Stacked_3D_and_2D_Regression_Networks.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ Figure 1. Document image unwarping. Top row: input images. Middle row: predicted
 ## How DewarpNet works?
 DewarpNet, as shown in Fig. 2, consists of two sub- networks for learning unwarping: the shape network and the texture mapping network. Additionally, a post-processing refinement module for illumination effect adjustment  visually improves the unwarped images.The training process has two phases. In the first phase, the shape network and the texture mapping network are trained separately for initialization. In the second phase, the two sub-networks are trained jointly to improve the unwarping result.They train the models on the Doc3D dataset of 100,000 images, splitting into training and validation sets such that they have no meshes in common. In the first phase of initialization training, the texture mapping network takes the ground truth 3D coordinate map C as input. Later, in the second phase of joint training, each sub-network is initialized with the best separately trained models.
 <p allign = "Center">
```
Comment (shivishbrahma) [inline DewarpNet_Single_Image_Document_Unwarping_With_Stacked_3D_and_2D_Regression_Networks.md]: Make this `<p align="center">`

**Review by shivishbrahma — CHANGES_REQUESTED** [review]
Check this one and request again

**Comment by shivishbrahma** [comment]
@Rajatava all is good but plz change

File: DewarpNet_Single_Image_Document_Unwarping_With_Stacked_3D_and_2D_Regression_Networks.md [hunk 2]
```diff
@@ -19,15 +19,15 @@ Re- cent work by Ma et al. proposed a deep learning system that directly regress
...
 
 
 ## How DewarpNet works?
 DewarpNet, as shown in Fig. 2, consists of two sub- networks for learning unwarping: the shape network and the texture mapping network. Additionally, a post-processing refinement module for illumination effect adjustment  visually improves the unwarped images.The training process has two phases. In the first phase, the shape network and the texture mapping network are trained separately for initialization. In the second phase, the two sub-networks are trained jointly to improve the unwarping result.They train the models on the Doc3D dataset of 100,000 images, splitting into training and validation sets such that they have no meshes in common. In the first phase of initialization training, the texture mapping network takes the ground truth 3D coordinate map C as input. Later, in the second phase of joint training, each sub-network is initialized with the best separately trained models.
-<p allign = "Center">
+<p allign = "Centre">
```
Comment (ucalyptus) [inline DewarpNet_Single_Image_Document_Unwarping_With_Stacked_3D_and_2D_Regression_Networks.md:30]: it should be `center` with a small c

File: DewarpNet_Single_Image_Document_Unwarping_With_Stacked_3D_and_2D_Regression_Networks.md [hunk 3]
```diff
@@ -19,15 +19,15 @@ Re- cent work by Ma et al. proposed a deep learning system that directly regress
 First, they contribute the Doc3D data-set. According to them, this is the first and largest document image data-set with multiple ground-truth annotations in both 3D and 2D domain.
 Second, they propose DewarpNet, a novel end-to-end deep learning architecture for document unwarping. This network enables high-quality document image unwarping in real-time.
 Third, trained with the rich annotations in the Doc3D data-set, DewarpNet shows superior performance compared to recent state-of-the-art . Evaluating with perceptual similarity to real document scans, it improves the multi-scale Structural Similarity (MS-SSIM) by 15% and reduce the Local Distortion by 36%. Furthermore, they demonstrate the practical significance of our method by a 42% decrease in OCR character error rate.
-<p allign = "Center">
+<p allign = "Centre">
```
Comment (ucalyptus) [inline DewarpNet_Single_Image_Document_Unwarping_With_Stacked_3D_and_2D_Regression_Networks.md:22]: `center`

**Review by ucalyptus — CHANGES_REQUESTED** [review]
spelling wrong . `center` hobe

**Pull request merged**